core/state/database.go

```diff
@@ -38,6 +38,8 @@ const (
 	codeCacheSize = 64 * 1024 * 1024
 )
 
+var TestVKTOpenStateless = false
```

Comment:
This is a test flag to force `OpenTrie(...)` open the underlying trie in stateless mode.
I'm not sure this flag will survive the final PR version. For now leaving it here, we can figure out that later.

Reply:
you don't need to do that, if your database isn't a `VerkleDB` then it should open an MPT.

Reply:
I've should mentioned to maybe wait until seeing the `BenchmarkTriesRandomVKTStateless(...)` below to have more context!